scripts/urban.js

```diff
@@ -0,0 +1,50 @@
+// Description
+//   Looks up a given phrase on Urban Dictionary
+//
+// Commands:
+//   `!urban <PHRASE>` - Looks a phrase up on Urban Dictionary.
+//
+
+module.exports = function (robot) {
+  robot.respond(/!?urban(.*)/i, function (res) {
+    var phrase = res.match[1].trim();
+    var urban = robot.http("http://api.urbandictionary.com/v0/define?term=" + encodeURI(phrase))
+      .get()(function (err, resp, body) {
+
+        // Check for correct usage
+        if (res.match[1][0] !== " " || !phrase || phrase.length < 1) {
```

Comment:
you won't need to check for whitespace if you fix the regex. Also, this if statement should be moved to before the `get()` request so we can validate it :)